03 Jul 2008 -- Kevin Quick <quick@sparq.org>

  * The General/ir file erroneously declares static variables as
    extern in the header file, which prevents compilation under later,
    stricter GCC implementations.

diff -Naur xmms-1.2.10.orig/General/ir/ir.h xmms-1.2.10/General/ir/ir.h
--- xmms-1.2.10.orig/General/ir/ir.h	2008-07-03 14:31:57.000000000 -0700
+++ xmms-1.2.10/General/ir/ir.h	2008-07-03 14:32:30.000000000 -0700
@@ -49,8 +49,8 @@
 }
 irConfig;
 
-extern pthread_t irapp_thread;
-extern gboolean keepGoing;
+/* extern pthread_t irapp_thread; */
+/* extern gboolean keepGoing; */
 extern irConfig ircfg;
 extern gboolean irconf_is_going;
 
